example/custom-fields/main.go

```diff
@@ -0,0 +1,173 @@
+// The simple command demonstrates the functionality that
+// prompts the user for a Clickup task and lists the values
+// of all custom fields for the given task.
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/raksul/go-clickup/clickup"
+)
+
+func main() {
+	api_key := os.Getenv("CLICKUP_API_KEY")
+	client := clickup.NewClient(nil, api_key)
+
+	task, _, err := client.Tasks.GetTask(context.Background(), "2v1etep", nil)
```

Comment:
```suggestion
	task, _, err := client.Tasks.GetTask(context.Background(), "taskId", nil)
```

Reply:
I have fixed the example code to get the taskID from stdin 💦